.github/workflows/release-and-publish-artifacts.yml

```diff
@@ -0,0 +1,33 @@
+name: release and push to central
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Similar question to below, any opposition to using @v3 here? -- https://github.com/actions/checkout/releases